src/wp-includes/class-wp-scripts.php

```diff
@@ -75,15 +75,6 @@ class WP_Scripts extends WP_Dependencies {
 	 */
 	public $do_concat = false;
 
-	/**
-	 * Whether the delayed inline script loader has been printed.
-	 *
-	 * @since 6.3.0
-	 * @see WP_Scripts::print_delayed_inline_script_loader()
-	 * @var bool
-	 */
-	public $printed_delayed_inline_script_loader = false;
```

Comment:
Not directly related to this line, but this PR should also remove the `delayed-inline-script-loader.js` file and any references to it.